tests/scripts/booking/cancel_old_unused_bookings_for_venue_test.py

```diff
@@ -0,0 +1,47 @@
+from datetime import datetime, timedelta
+import pytest
+from pcapi.core.bookings.models import Booking
+
+from pcapi.model_creators.generic_creators import create_booking, create_deposit, create_offerer, create_user, create_venue
+from pcapi.repository import repository
+from pcapi.scripts.booking.cancel_old_unused_bookings_for_venue import cancel_old_unused_bookings_for_venue
+from pcapi.utils.human_ids import humanize
+
+
+@pytest.mark.usefixtures("db_session")
+def test_should_cancel_old_unused_bookings_for_venue():
+    venue_id = 10
+    offerer = create_offerer()
+    venue = create_venue(idx=venue_id, offerer=offerer)
+    beneficiary = create_user()
+    create_deposit(user=beneficiary)
+
+    to_cancel_booking_id = 11
+    used_booking_id = 12
+    recent_booking_id = 13
+
+    to_cancel_booking = create_booking(date_created=(
+        datetime.now() - timedelta(days=40)), user=beneficiary, venue=venue, idx=to_cancel_booking_id)
+
+    used_booking = create_booking(date_created=(
+        datetime.now() - timedelta(days=40)), user=beneficiary, venue=venue, idx=used_booking_id, is_used=True)
+
+    recent_booking = create_booking(date_created=(
+        datetime.now() - timedelta(days=10)), user=beneficiary, venue=venue, idx=recent_booking_id)
+
+    repository.save(to_cancel_booking, used_booking, recent_booking)
```

Comment:
Désormais, mieux vaut utiliser les *factories* pour les nouveaux tests. Ça donnerait ça (pas besoin d'appeler `repository.save()`) :

```suggestion
    import pcapi.core.bookings.factories as bookings_factories
    import pcapi.core.offers.factories as offers_factories

    venue  = offers_factories.VenueFactory()
    to_cancel_booking = BookingFactory(
        dateCreated=datetime.now() - timedelta(days=40),
        stock__offer__venue=venue,
    )
    used_booking = BookingFactory(
        dateCreated=datetime.now() - timedelta(days=40),
        isUsed=True,
        stock__offer__venue=venue,
    )
    recent_booking = BookingFactory(
        dateCreated=datetime.now() - timedelta(days=10),
        stock__offer__venue=venue,
    )
```